VMEncryption/main/handle.py

```diff
@@ -270,6 +271,89 @@ def get_protected_settings():
     else:
         return protected_settings_str
 
+def update_encryption_settings_luks2_header(extra_items_to_encrypt=[]):
+    '''This function is used for CMK passphrse wrapping with new KEK URL and update metadata in LUKS2 header.'''
+    hutil.do_parse_context('UpdateEncryptionSettingsLuks2Header')
+    logger.log('Updating encryption settings LUKS-2 header')
+    # ensure cryptsetup package is still available in case it was for some reason removed after enable
+    try:
+        DistroPatcher.install_cryptsetup()
```

Comment:
Installation of cryptsetup must be in the pre-req checks per distro? That's during ADE installation.

Reply:
install_cryptsetup checks if package is installed, it installs only if package is not present.